### PR TITLE
Change the way we obtain the Wazuh revision

### LINKED ISSUE #7522: Update get_api_version function from orm.py
Hello team,

Currently the `orm.py` gets the Wazuh version (used to check the integrity of the rbac.db) by using the `get_api_version` function which reads the value from the `spec.yml`. As Core team introduced a new function to obtain the Wazuh version we should addapt our `get_api_version` function to use it instead of getting the value by reading the Spec.yaml.

### PR DESCRIPTION
**Pull request by CarlosRS9**
Hello team,

This PR closes #7522. The `DatabaseManager.get_api_revision` function from the `orm` has been removed. The current API revision will be obtained from `common.get_wazuh_revision` instead, which uses `wazuh-control info` to obtain this information instead of having to load the `spec.yaml` to get this information.

To ensure we have the right permissions to read the Wazuh revision from the `wazuh-control` we had call this function before dropping the privileges from `root` to `ossec` in `wazuh-apid`.

**Note: Jenkins integration tests will fail until the #8037 is merged.** This is not related to the changes introduced in this PR but a problem with the dependencies.

## Tests results
**Unit tests**
```
============================================= test session starts ==============================================
platform linux -- Python 3.8.5, pytest-5.0.0, py-1.9.0, pluggy-0.13.1
rootdir: /home/carlos/GIT/wazuh/api
plugins: metadata-1.11.0, tavern-1.0.0, html-3.1.1, testinfra-5.0.0
collected 225 items                                                                                            

api/api/test/test_alogging.py ..........                                                                 [  4%]
api/api/test/test_authentication.py ..........                                                           [  8%]
api/api/test/test_configuration.py .........................................                             [ 27%]
api/api/test/test_util.py ...................................                                            [ 42%]
api/api/test/test_validator.py ......................................................................... [ 75%]
........................................................                                                 [100%]

=================================== 225 passed, 17 warnings in 2.22 seconds ====================================
```

```
============================================= test session starts ==============================================
platform linux -- Python 3.8.5, pytest-5.0.0, py-1.9.0, pluggy-0.13.1
rootdir: /home/carlos/GIT/wazuh/framework
plugins: metadata-1.11.0, tavern-1.0.0, html-3.1.1, testinfra-5.0.0
collected 228 items                                                                                            

framework/wazuh/rbac/tests/test_auth_context.py ..                                                       [  0%]
framework/wazuh/rbac/tests/test_decorators.py .......................................................... [ 26%]
...............................................                                                          [ 46%]
framework/wazuh/rbac/tests/test_default_configuration.py ............................................... [ 67%]
........                                                                                                 [ 71%]
framework/wazuh/rbac/tests/test_orm.py .......................................................           [ 95%]
framework/wazuh/rbac/tests/test_preprocessor.py ...........                                              [100%]
=================================== 228 passed, 3 warnings in 440.14 seconds ===================================
```
```
============================================= test session starts ==============================================
platform linux -- Python 3.8.5, pytest-5.0.0, py-1.9.0, pluggy-0.13.1
rootdir: /home/carlos/GIT/wazuh/framework
plugins: metadata-1.11.0, tavern-1.0.0, html-3.1.1, testinfra-5.0.0
collected 789 items                                                                                            

framework/wazuh/tests/test_active_response.py ............                                               [  1%]
framework/wazuh/tests/test_agent.py .................................................................... [ 10%]
............................                                                                             [ 13%]
framework/wazuh/tests/test_cdb_list.py .....................................................             [ 20%]
framework/wazuh/tests/test_ciscat.py .................................                                   [ 24%]
framework/wazuh/tests/test_cluster.py .....ss                                                            [ 25%]
framework/wazuh/tests/test_decoder.py ...................................                                [ 29%]
framework/wazuh/tests/test_group.py ........                                                             [ 30%]
framework/wazuh/tests/test_logtest.py ......                                                             [ 31%]
framework/wazuh/tests/test_manager.py .................................                                  [ 35%]
framework/wazuh/tests/test_mitre.py .................................................................... [ 44%]
........................................................................................................ [ 57%]
........                                                                                                 [ 58%]
framework/wazuh/tests/test_rootcheck.py .................................................                [ 64%]
framework/wazuh/tests/test_rule.py ........................................................              [ 71%]
framework/wazuh/tests/test_sca.py .......                                                                [ 72%]
framework/wazuh/tests/test_security.py ................................................................. [ 81%]
...........................................                                                              [ 86%]
framework/wazuh/tests/test_stats.py ................                                                     [ 88%]
framework/wazuh/tests/test_syscheck.py ........................                                          [ 91%]
framework/wazuh/tests/test_syscollector.py ............                                                  [ 93%]
framework/wazuh/tests/test_task.py ............................                                          [ 96%]
framework/wazuh/tests/test_vulnerability.py ..........................                                   [100%]
============================= 787 passed, 2 skipped, 15 warnings in 157.78 seconds =============================
```

**Integration tests**
```
test_cluster_endpoints.tavern.yaml 
	 44 passed, 1 xpassed, 1 warnings

test_manager_endpoints.tavern.yaml 
	 31 passed, 1 warnings

test_rbac_black_cluster_endpoints.tavern.yaml 
	 17 passed, 1 warnings

test_rbac_black_manager_endpoints.tavern.yaml 
	 15 passed, 1 warnings

test_rbac_black_security_endpoints.tavern.yaml 
	 25 passed, 1 warnings

test_rbac_white_cluster_endpoints.tavern.yaml 
	 17 passed, 1 warnings

test_rbac_white_manager_endpoints.tavern.yaml 
	 15 passed, 1 warnings

test_rbac_white_security_endpoints.tavern.yaml 
         25 passed, 1 warnings

test_security_DELETE_endpoints.tavern.yaml 
	 15 passed, 1 warnings

test_security_GET_endpoints.tavern.yaml 
	 18 passed, 1 warnings

test_security_POST_endpoints.tavern.yaml 
	 7 passed, 1 warnings

test_security_PUT_endpoints.tavern.yaml 
	 9 passed, 1 warnings
```

### CODE REVIEW

**Review by davidjiglesias — CHANGES_REQUESTED** [review]
common.get_wazuh_revision() can return  "ERROR", I would like to handle this case. We can print a message in API log saying there was an error obtaining current version, and the database will remain unchanged for this restart.

**Review by vicferpoy — APPROVED** [review]
LGTM!

File: api/scripts/wazuh-apid.py [hunk 1]
```diff
@@ -130,9 +133,6 @@ def start(foreground, root, config_file):
     else:
         print(f"Starting API in foreground")
 
-    # Load the SPEC file into memory to use as a reference for future calls
```
Comment (davidjiglesias) [inline api/scripts/wazuh-apid.py:133]: Do not remove

**Review by davidjiglesias — CHANGES_REQUESTED** [review]

File: api/scripts/wazuh-apid.py (hunk 1 above)
Comment (CarlosRS9) [reply api/scripts/wazuh-apid.py]: Restored

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**